ingest/config/config.go

```diff
@@ -226,21 +228,36 @@ func (ic *IngestConfig) Verify() error {
 		return err
 	}
 
-	// Make sure the log directory exists.
-	logdir := filepath.Dir(ic.Log_File)
-	fi, err := os.Stat(logdir)
-	if err != nil {
-		if os.IsNotExist(err) {
-			//try to make the directory
-			err = os.MkdirAll(logdir, 0700)
-			if err != nil {
+	if ic.Log_UDP_Target != `` {
+		//make sure we can do a valid host/port split and the port is > 0 and <= 0ffff
+		if host, port, err := net.SplitHostPort(ic.Log_UDP_Target); err != nil {
+			return fmt.Errorf("Invalid Log-UDP-Target %q %w", ic.Log_UDP_Target, err)
+		} else if host == `` {
+			return fmt.Errorf("Invalid Log-UDP-Target %q missing host", ic.Log_UDP_Target)
+		} else if port == `` {
+			return fmt.Errorf("Invalid Log-UDP-Target %q missing port", ic.Log_UDP_Target)
+		} else if p, err := strconv.ParseUint(port, 10, 16); err != nil {
+			return fmt.Errorf("Invalid Log-UDP-Target %q port %s is invalid %w", ic.Log_UDP_Target, port, err)
+		} else if p <= 0 || p > 0xffff {
+			return fmt.Errorf("Invalid Log-UDP-Target %q port %s is invalid", ic.Log_UDP_Target, port)
+		}
+	} else {
```

Comment:
This looks like we skip the log file checks if the UDP logger is configured. Are they mutually exclusive?

Reply:
correct and yes.